### PR TITLE
PR for Social Media Screens

### PR DESCRIPTION
**Pull request by manojasarsa**
This PR includes screens for social media that includes:

- Signin and Signup
- Home 
- Explore
- Notifications
- Profile
- Bookmarks

### CODE REVIEW

File: src/pages/authentication/Signin.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        <div className="flex justify-items-center items-center">
+            <div className="m-16 mx-auto w-full ">
+                <div className="md:container md:mx-auto mx-auto flex">
+                    <img src="assets/social1.svg" alt="social-img" className="w-1/2 hidden md:block "></img>
+                    <div className="mx-auto sm:mx-28 mt-16">
+
+                        <div className="flex flex-col mx-auto pb-8 shadow-none border-2 min-w-max ">
+
+                            {/* <h1 className="mx-auto font-serif text-5xl p-2 ">Alcon</h1> */}
```
Comment (Chakravarthi0) [inline src/pages/authentication/Signin.jsx:31]: You can remove this comment if not needed

File: src/pages/authentication/Signin.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+                                </label>
+
+                                {/* TODO */}
+
+                                {/* <button className="my-3 text-x cursor-pointer text-center py-1 border-2 font-semibold  text-slate-900 bg-sky-300" onClick={(e) => formSignIn(e)} >Login</button> */}
+
+                                <Link to="/home" className="my-3 text-x cursor-pointer text-center py-1 border-2 font-semibold  bg-blue-600 hover:bg-blue-700 text-white">Login</Link>
+
+                                {/* <button className="my-3 text-x cursor-pointer text-center py-1 border-2 font-semibold text-sky-500" onClick={(e) => formGuestSignIn(e)} >Guest Login</button> */}
```
Comment (Chakravarthi0) [inline src/pages/authentication/Signin.jsx:70]: I think you forgot to remove these comments

File: src/pages/authentication/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                }
+        } else {
+                console.log("error");
+        }
+    }
+    
+    return (
+            <div className="flex flex-col pb-1 shadow-none border-2 w-1/4 mx-auto mt-12 min-w-max">
+
+                {/* <h1 className="mx-auto font-serif text-5xl p-2 ">Alcon</h1> */}
```
Comment (Chakravarthi0) [inline src/pages/authentication/Signup.jsx:42]: remove this comment, if not needed

**Review by Chakravarthi0 — APPROVED** [review]
UI looks great.
there are few comments which can be removed, except that, all good

**Review by pritam-kr — APPROVED** [review]
- The UI looks clean.
- clean code.
- Good folder structure.

**Pull request merged**